modules/osd-monitoring-configurable-monitoring-components.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-configuring-the-monitoring-stack.adoc
+
+[id="configurable-monitoring-components_{context}"]
+= Configurable monitoring components
+
+This table shows the monitoring components you can configure and the keys used to specify the components in the `user-workload-monitoring-config` `ConfigMap` objects:
+
+.Configurable monitoring components
+[options="header"]
+|====
+//|Component |cluster-monitoring-config ConfigMap key |user-workload-monitoring-config config map key
```

Comment:
should show the title, remove `//`

Reply:
Good catch. I have removed the column for `cluster-monitoring-config ConfigMap key`